src/tools/rbd/action/Info.cc

```diff
@@ -136,7 +148,9 @@ static int do_show_info(const char *imgname, librbd::Image& image,
               << "\tblock_name_prefix: " << prefix
               << std::endl
               << "\tformat: " << (old_format ? "1" : "2")
-              << std::endl;
+	      << std::endl
+	      << "\tconsistency group: " << group_string
```

Comment:
Minor: I'd leave it out of the output if there is no consistency group (like we leave out the journal or mirroring fields).